src/i_video.c

```diff
@@ -1581,7 +1581,6 @@ static void I_InitGraphicsMode(void)
     uint32_t flags = 0;
 
     // [FG] window flags
-    flags |= SDL_WINDOW_RESIZABLE;
```

Comment:
Something like this is needed to allow the window to be resized when toggling between fullscreen (borderless) and windowed:

```diff
diff --git a/src/i_video.c b/src/i_video.c
index 56a721b5..888a8217 100644
--- a/src/i_video.c
+++ b/src/i_video.c
@@ -397,6 +397,7 @@ static void I_ToggleFullScreen(void)
     {
         flags |= SDL_WINDOW_FULLSCREEN_DESKTOP;
         SDL_SetWindowGrab(screen, SDL_TRUE);
+        SDL_SetWindowResizable(screen, SDL_FALSE);
     }
 
     SDL_SetWindowFullscreen(screen, flags);
@@ -408,6 +409,7 @@ static void I_ToggleFullScreen(void)
     {
         SDL_SetWindowGrab(screen, SDL_FALSE);
         AdjustWindowSize();
+        SDL_SetWindowResizable(screen, SDL_TRUE);
         SDL_SetWindowSize(screen, window_width, window_height);
     }
 }
```

Reply:
Added that, thanks.

Related to borderless mode, on this branch do you notice any differences in responsiveness when starting in borderless mode, then alt-entering out of then back into fullscreen? I do on my Win10 machine (with Game Bar and other overlays disabled). This is fixed if I change `I_InitGraphicsMode` to create the window as non-fullscreen and set `toggle_fullscreen=true` to switch it to fullscreen mode in the first update.